### PR TITLE
Python Docker Container Actions via pipenv

### PR DESCRIPTION
**Pull request by NaveenKundeti-IBS**
Python Docker Github Actions via pipenv

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM python:3.8-slim-buster
+WORKDIR /chatops
+COPY . .
+RUN chmod +x script.sh
+CMD cd /chatops && ./script.sh
```
Comment (tomegathericon) [inline Dockerfile:5]: @NaveenKundeti-IBS  Please `cd` once either in the `Dockerfile` or the `script`

Comment (NaveenKundeti-IBS) [reply Dockerfile:5]: yes updated in the script file sir @tomegathericon

File: script.sh [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+pip install pipenv
+pipenv install && pipenv run python main.py
```
Comment (tomegathericon) [inline script.sh]: @NaveenKundeti-IBS  Since this is a shell script please have all steps in separate lines

Comment (NaveenKundeti-IBS) [reply script.sh]: Updated Sir @tomegathericon as per your suggestion.

**Pull request merged**